packages/flutter/lib/src/material/input.dart

```diff
@@ -54,18 +51,21 @@ class Input extends StatefulWidget {
     this.isDense: false,
     this.autofocus: false,
     this.maxLines: 1,
-    this.formField,
     this.onChanged,
     this.onSubmitted,
   }) : super(key: key);
 
-  /// The text of the input field.
-  ///
-  /// If the [Input] is in a [Form], this is the initial value only.
-  ///
-  /// Otherwise, this is the current value, and must be updated every
-  /// time [onChanged] is called.
-  final InputValue value;
+  /// The editable text displayed in the input field. If this is null, the
+  /// [Input] manages it for you.
```

Comment:
this should discuss all the implications of setting it and not (e.g. that setting it to a value that doesn't match the state causes the selection to be blown away, etc)